spec/state_machines/global_state_machine_spec.rb

```diff
@@ -107,7 +140,7 @@
         [:press_officer, :full_unassigned_foi],
         [:press_officer, :full_awresp_foi],
         [:press_officer, :full_draft_foi],
-      )
+      ).debug
```

Comment:
Do we need the debugs left in?

Reply:
Don't need them and I'll take them out, but generally they shouldn't cause an issue as they don't activate if the file is being run at a terminal